presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -56,18 +56,34 @@ Configuration Properties
 
 The following configuration properties are available:
 
-========================================================== ==============================================================================
+========================================================== ==============================================================
 Property Name                                              Description
-========================================================== ==============================================================================
+========================================================== ==============================================================
 ``kafka.table-names``                                      List of all tables provided by the catalog
 ``kafka.default-schema``                                   Default schema name for tables
 ``kafka.nodes``                                            List of nodes in the Kafka cluster
 ``kafka.buffer-size``                                      Kafka read buffer size
 ``kafka.table-description-dir``                            Directory containing topic description files
+``kafka.timestamp-upper-bound-force-push-down-enabled``    Controls if upper bound timestamp push down is enabled for topics using ``CreateTime`` mode
 ``kafka.hide-internal-columns``                            Controls whether internal columns are part of the table schema or not
 ``kafka.messages-per-split``                               Number of messages that are processed by each Presto split, defaults to 100000
-``kafka.timestamp-upper-bound-force-push-down-enabled``    Controls if upper bound timestamp push down is enabled for topics using ``CreateTime`` mode
-========================================================== ==============================================================================
+``kafka.security.protocol``                                Security protocol for connection to Kafka cluster, defaults to ``PLAINTEXT``
```

Comment:
It would be great to have a section for each parameter below. 
In the table I would drop the default value information and move it to per-config-parameter sections below.

Reply:
will do